tests/test_choices.py

```diff
@@ -271,7 +272,7 @@ def __init__(self):
         except ValueError as v:
             self.assertEqual(
                 str(v),
-                "Invalid value 'invalid' for property 'prop'; Valid values are: a, b, none"
+                "Invalid value 'invalid' for property 'prop'; Valid values are: a, b, none, none"
```

Comment:
This makes me *very* skeptical. Why would we want to allow both 'none' as a string value, and None as a symbolic constant, rendered as a string indistinguishable from the string form?

Reply:
This is a side effect of what you're calling the "original mistake." When making the proposed change, the ValueError's message also coerces Python `None` to the string `'none'`. So, since we are allowing both `None` and `'none'` as values in the proposed change, this is what the message ends up looking like (repeating `'none'` twice).